### PR TITLE
[Arista] Add port definitions for 7060X6 HwSkus in port_utils

### PR DESCRIPTION
**Pull request by Staphylo**
### Description of PR

Summary: Add port definition for Arista 7060X6 HwSkus in port_utils

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [x] 202311

#### What is the motivation for this PR?

Allow add-topo to properly generate a minigraph on these new 7060X6 HwSku
Also register these HwSku into the `broadcom_<asic>_hwskus` to allow some tests to pass.

#### How did you do it?

Added the definitions of the 7060X6 HwSkus in the `port_utils.py` and `variables` files

#### How did you verify/test it?

Ran t0 and t1 topologies on these HwSkus

#### Any platform specific information?

Specific to Arista 7060X6

### CODE REVIEW

**Review by prgeor — APPROVED** [review]

**Review by r12f — APPROVED** [review]

**Comment by r12f** [comment]
I have tried the port config file and some I am hitting this issue before due to Ethernet2 key is missing in the hwsku_dict:

image

and the hwsku.json looks like below:
image

@ccroy-arista , do you mind to help see if anything is missing and related to this change? Thanks a lot!

**Comment by Staphylo** [comment]
@r12f this issue is unrelated to this change so this one should be ready to go.

You are running into the subport issue.
We actually delayed publishing our `hwsku.json` until the subport fix merged so you wouldn't run into this issue.
However it seems that your image has the `hwsku.json` change but not the subport fix.
Please make sure you have both these in your image.
https://github.com/sonic-net/sonic-buildimage/pull/18499
https://github.com/sonic-net/sonic-swss/pull/3161

**Comment by r12f** [comment]
Hi Samuel, is the build image change back ported to 202311?

**Comment by r12f** [comment]
after deploying this change, deploying minigraph is not failing anymore.

**Review by r12f — APPROVED** [review]
lgtm. consider approved after fixing the CI.

**Review by Pterosaur — APPROVED** [review]

**Comment by Pterosaur** [comment]
@yxieca @bingwang-ms  Please help to approve the request labels.

**Comment by Janetxxx** [comment]
@yxieca @bingwang-ms Please help to approve the request labels.

**Pull request merged**
